extensions/wikia/AssetsManager/config.php

```diff
@@ -899,6 +900,7 @@
 		'//extensions/wikia/AdEngine/js/video/videoFrequencyMonitor.js',
 		'//extensions/wikia/AdEngine/js/video/videoFrequencySettings.js',
 		'//extensions/wikia/AdEngine/js/video/videoFrequencyStore.js',
+		'//extensions/wikia/ArticleVideo/scripts/featured-video.cookies.js',
```

Comment:
Is it fine for mobile-wiki? There is a bit different implementation of jwplayer. Check how we set autoplay flag there please:
https://github.com/Wikia/mobile-wiki/blob/dev/app/modules/video-players/jwplayer.js